src/modules/entity/reducer.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { EntityType, EntityVersion } from 'dcl-catalyst-commons'
+import {
+  fetchEntitiesByIdsFailure,
+  fetchEntitiesByIdsRequest,
+  fetchEntitiesByIdsSuccess,
+  fetchEntitiesByPointersFailure,
+  fetchEntitiesByPointersRequest,
+  fetchEntitiesByPointersSuccess
+} from './actions'
+import { EntityState, entityReducer } from './reducer'
+
+const entity = {
+  id: 'Qmhash',
+  timestamp: 1234,
+  type: EntityType.WEARABLE,
+  pointers: ['aPointer'],
+  content: [
+    {
+      hash: 'Qmhash',
+      file: 'pepito.jpg'
+    }
+  ],
+  metadata: {
+    owner: '0xpepito',
+    some: 'thing'
+  },
+  version: EntityVersion.V3
+}
+
+describe('when reducing the FETCH_ENTITIES_BY_POINTERS_REQUEST action', () => {
+  it('should add an action to the loading state', () => {
+    const state: EntityState = {
+      data: {},
+      loading: [],
+      error: null
+    }
+    const action = fetchEntitiesByPointersRequest(EntityType.WEARABLE, entity.pointers)
+    const newState = entityReducer(state, action)
+    expect(newState.loading).toHaveLength(1)
+  })
+})
+
+describe('when reducing the FETCH_ENTITIES_BY_POINTERS_SUCCESS action', () => {
+  it('should insert the entity into the state data and clear the loading state', () => {
+    const state: EntityState = {
+      data: {},
+      loading: [fetchEntitiesByPointersRequest(EntityType.WEARABLE, entity.pointers)],
+      error: null
+    }
+
+    const action = fetchEntitiesByPointersSuccess(EntityType.WEARABLE, entity.pointers, [entity])
+    const newState = entityReducer(state, action)
+    expect(newState.data[entity.id]).toEqual(entity)
+    expect(newState.loading).toHaveLength(0)
```

Comment:
These expectations are fine, but I would have checked the entire data structure to be sure that it added just one with the id we wanted (it's a bit more robust).

Reply:
done 👍